### ISSUE TITLE
Modify ansible scripts to work with latest travis image

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/purl.obolibrary.org#337]
Various PRs were failing, e.g. #335 

We temporarily addressed this with #336 

__this is a temporary fix__

It looks like we will have to modify the ansible yml, e.g.

```
38.58s$ ansible-playbook -i hosts -e 'mode=travis' --connection=local site.yml
[DEPRECATION WARNING]: Instead of sudo/sudo_user, use become/become_user and 
make sure become_method is 'sudo' (default).
This feature will be removed in a 
future release. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
```

**Comment by cmungall** [comment]
From https://blog.travis-ci.com/2017-06-21-trusty-updates-2017-Q2-launch

__We are planning to keep the previous generation of images around until September 1st, 2017__

**Comment by kltm** [comment]
I believe that this may be due to the 1.9+ change to `become` from `sudo/su`.
http://docs.ansible.com/ansible/become.html
We went through this transition a bit back for monarch and go; a bit more verbose (IMO).

**Comment by jamesaoverton** [comment]
I made a [travis-update](https://github.com/OBOFoundry/purl.obolibrary.org/tree/travis-update) branch, and it's running OK now: https://travis-ci.org/OBOFoundry/purl.obolibrary.org/builds/252518848

The problem seems to be isolated to the apt cache update part of the Ansible script, so I suspect this longstanding Ansible bug: https://github.com/ansible/ansible-modules-core/issues/2951

By disabling the apt update our packages could be stale, but that's probably not a big deal. An alternative is to update apt with a shell command.

A second opinion would be appreciated.

**Comment by kltm** [comment]
We've had a lot of headaches recently mixing docker and ansible. While a little frustrating that we cannot keep the orchestration centralized in the ansible files, we've started looking at it as a two step process: create an up-to-date docker image, including caches, use ansible to get it into functional state.
Not completely analogous to the issues with the images used by travis, but there seems to be a continuing series of frustrations in mixing the two; and the only solution seems to often be just pushing that part back into docker/travis.

Personally, I would be a little squeamish about disabling the package updates.

**Comment by jamesaoverton** [comment]
I switched to a shell command for `apt-get update` and created PR #341. I'm happy with this solution for the medium term.

**Comment by jamesaoverton** [comment]
Closed by #341
